doc/_themes/ceph/static/nature.css_t

```diff
@@ -106,92 +106,99 @@ div.sphinxsidebar h4 {
 div.sphinxsidebar h4{
     font-size: 1.1em;
 }
- 
+
 div.sphinxsidebar h3 a {
     color: #e6e8e8;
 }
- 
- 
+
+div.sphinxsidebar h4 a {
```

Comment:
nit, instead of repeating the code setting, you could put something like
```css
div.sphinxsidebar h3 a,
div.sphinxsidebar h4 a {
    color: #e6e8e8;
}
```

Reply:
Can do